lonboard/__main__.py

```diff
@@ -0,0 +1,3 @@
+from lonboard._cli import main
+
+main()
```

Comment:
not sure why we need this

Reply:
This enables `python -m lonboard file.geojson`